lib/deploymentTool.js

```diff
@@ -0,0 +1,24 @@
+const BASE = "cli-firebase";
+
+function _value() {
+  if (!process.env.FIREBASE_DEPLOY_AGENT) {
```

Comment:
I don't think the if statement is necessary. Line 8 covers both cases.

Reply:
I don't think so...

    ['abc',undefined].join('--'); // -> abc--

Reply:
Oh yeah, you're right.